nixos/modules/virtualisation/amazon-config.nix

```diff
@@ -1,5 +1,3 @@
-{ config, pkgs, modulesPath, ... }:
-
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-image.nix" ];
+  imports = [ ./amazon-image.nix ];
```

Comment:
ok, I guess I understand now.

When we include amazon-image.nix, then we produce an amazon image which is running an installation script.  The default configuration then copy this file over to be the default configuration of the installed system.

I will fix it as you suggested now, but I think that this should be somehow part of the hardware detection script, and also that this is sad that the original configuration of the image is not carried over to the installed configuration.